### PR TITLE
Refresh command

### PR DESCRIPTION
**Pull request by limenet**

### CODE REVIEW

File: src/Command/Refresh.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            self::OPTION_ASSETS => Asset::class,
+            self::OPTION_DOCUMENTS => Document::class,
+            self::OPTION_OBJECTS => Concrete::class,
+        };
+
+        foreach ($this->input->getOption($optionName) as $id) {
+            $this->output->writeln($id);
+            $element = $objClass::getById($id);
+            if ($element === null) {
+                $this->output->writeln('-> not found, skipped');
```
Comment (pryserv) [inline src/Command/Refresh.php:103]: What do you think about adding the ID and type ($objJClass) of the not found element?

File: src/Command/Refresh.php [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        $objClass = match ($optionName) {
+            self::OPTION_ASSETS => Asset::class,
+            self::OPTION_DOCUMENTS => Document::class,
+            self::OPTION_OBJECTS => Concrete::class,
+        };
+
+        foreach ($this->input->getOption($optionName) as $id) {
+            $this->output->writeln($id);
+            $element = $objClass::getById($id);
+            if ($element === null) {
```
Comment (pryserv) [inline src/Command/Refresh.php:102]: I would add a blank line

**Review by pryserv — APPROVED** [review]
Nice enhancement 👍

File: src/Command/Refresh.php (hunk 1 above)
Comment (limenet) [reply src/Command/Refresh.php]: good idea, thx

File: src/Command/Refresh.php (hunk 2 above)
Comment (limenet) [reply src/Command/Refresh.php:103]: thanks for catching that

**Pull request merged**